tests/test_plugin.py

```diff
@@ -107,27 +107,9 @@ def test_plugin_disable(node_factory):
         n.rpc.hello(name='Sun')
 
 
-def test_plugin_hook(node_factory, executor):
```

Comment:
This patch is messed up?  Seems to remove existing plugin tests....

Reply:
This is my last rebase I made a mistake...